tripFrontend/src/features/auth/components/Input/Input.tsx

```
@@ -0,0 +1,42 @@
+
+import { JSX } from 'react';
+
+import { cn } from '@/lib/cn';
+
+type Props = {
+	className?: string;
+	value?: React.RefObject<HTMLInputElement | null>;
+	label?: string;
+	error?: string;
+	type?: string;
+	[key: string]: string | object | void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)